callstack/react-native-builder-bob
# feat: don't add a XCode prebuild action to invoke codegen anymore

### Summary

We used to add an xcode prebuild script to invoke codegen each time the user built their example application. However, with the latest setup, the action just causes the build to fail immediately. This PR removes that script. With this removal, the user has to install pods each time they change their codegen specs.

### Test plan

1. Create a new library using `crnl` and make sure it supports the new architecture
2. Install the pods and try building the example app using xcode
3. Make sure the build doesn't fail immediately.
